README.md

```diff
@@ -1,32 +1,54 @@
-## uk.co.vedaconsulting.mosaico
+# uk.co.vedaconsulting.mosaico
 This extension integrates Mosaico a responsive email template editor, with CiviCRM.
-- Initial Blog Post - https://civicrm.org/blogs/parvez/a-new-beginning-for-civimail
-- Beta Blog Post - https://civicrm.org/blog/deepaksrivastava/email-template-builder-civimosaico-is-now-beta
-- Initial Video - https://vimeo.com/156633077
 
-### Requirements
+ * [Initial Blog Post](https://civicrm.org/blogs/parvez/a-new-beginning-for-civimail)
+ * [Beta Blog Post](https://civicrm.org/blog/deepaksrivastava/email-template-builder-civimosaico-is-now-beta)
+ * [Initial Video](https://vimeo.com/156633077)
+ * [v2.0 Plans](https://civicrm.org/blog/jamienovick/email-template-builder-mosaico-phase-2-plans)
+ * [v2.0 and Styling Blog Post](https://civicrm.org/blog/jamienovick/extreme-makeovers-civicrm-style-introducing-the-shoreditch-theme-civicrms-new-user)
+
+## Requirements
+
+ * CiviCRM v4.7.16+ / (unofficial) CiviCRM v4.6.26+ with [backports patch #9555](https://github.com/civicrm/civicrm-core/pull/9555) applied
 
- * CiviCRM v4.7.16+ (or v4.6.26+ with unofficial backports from [#9555](https://github.com/civicrm/civicrm-core/pull/9555))
- * A CiviCRM extensions folder (In new sites since CiviCRM v4.7.0, this defaults to `files/civicrm/ext`. For older systems, see [the wiki](https://wiki.civicrm.org/confluence/display/CRMDOC/Extensions).)
  * PHP-ImageMagick
- * (For CLI-based installation) [`cv`](https://github.com/civicrm/cv)
- * (For git-based installation) [`cv`](https://github.com/civicrm/cv), [`git`](https://git-scm.com/), and [`grunt-cli`](http://gruntjs.com/getting-started)
 
-### Installation (CLI)
 
-If you have the command line tool [`cv`](https://github.com/civicrm/cv), then download Mosaico and related extensions:
+## Installation
+A CiviCRM extensions folder (In new sites since CiviCRM v4.7.0, this defaults to `files/civicrm/ext`. For older systems, see [the wiki](https://wiki.civicrm.org/confluence/display/CRMDOC/Extensions).)
+
+#### Option 1: CiviCRM.org Extension Directory
+This option does not require any extra server side dependency. To intall using this option, simply download the latest versions of all three extensions below and install them in order:
+
+1. [`Shoreditch Theme`](https://civicrm.org/extensions/shoreditch)
+2. [`FlexMailer`](https://civicrm.org/extensions/flexmailer)
+3. [`Mosaico CiviCRM Integration`](https://civicrm.org/extensions/email-template-builder)
```

Comment:
it is mentioned above to download the latest version for all three extensions. If a user download them from CiviCRM.org, he/she should not need to concern git branch. Think of a ordinary user.